Error: Cannot find module 'es2015-i18n-tag'

## What did you do

Upgrade to `4.0.9`.

## What did you expect to happen

Running without error

## What happened

**Error: Cannot find module 'es2015-i18n-tag'**
```sh
npm run test

> cyclic-schemas-test@1.0.0 test .../dev/cyclic-schemas-test
> jsonschema2md -o docs/ -x docs/ -d schemas/ -e json

internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'es2015-i18n-tag'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (.../dev/cyclic-schemas-test/node_modules/@adobe/jsonschema2md/lib/index.js:23:24)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

## What's your environment

* Operating System: macOS Catalina `10.15.2 (19C57)`
* node.js version: `v10.15.0`

## Do you have example files:

PR: https://github.com/axa-ch/cyclic-schemas-test/pull/2/files#r368655818

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
   },
   "dependencies": {
     "@adobe/helix-log": "^4.4.0",
-    "es2015-i18n-tag": "^1.6.1",
+    "es2015-i18n-tag": "1.6.1",
     "ferrum": "^1.4.1",
     "fs-extra": "^8.1.0",
     "github-slugger": "^1.2.1",

